src/test/rustdoc/playground-none.rs

```diff
@@ -1,3 +1,5 @@
+// compile-flags: --playground-url="" -Z unstable-options
```

Comment:
This test now ensures that we can disable it by providing an empty value to `playground-url` option.